bin/sk-tests

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-karma start --single-run "$@"
+sk-lint && karma start --single-run "$@"
```

Comment:
If we have `sk-lint` rule before commit then putting it here seems like an overkill. Very annoying overkill :)